### PR TITLE
Adding PipelineExecutor paragraph to the upgrade guide

### PR DESCRIPTION
**Pull request by HEskandari**
Related to https://github.com/Particular/NServiceBus/issues/3256

### CODE REVIEW

File: nservicebus/upgrades/5to6.md [hunk 1]
```diff
@@ -241,6 +241,9 @@ The `IMessageMutator` and `IMutateTransportMessages` interfaces are no longer av
  * `IncomingContext` and `OutgoingContext` have been replaced with multiple contexts ("stages") per pipeline offering more useful information depending on the requirements. The [Version 6 guide on pipeline customization](/nservicebus/pipeline/customizing-v6.md) provides more details about the available stages.
  * `Behavior<TContext>` now provides the base class for implementing custom behaviors and replaces `IBehavior<TContext>`.
 
+### PipelineExecutor
+The `PipelineExecutor` has been removed. This has been internalized and is not accessible through the Container anymore. If you are relying on it in your code, you can leverage a Behavior. The [Version 6 guide on pipeline customization](/nservicebus/pipeline/customizing-v6.md) provides more details about custom behaviors.
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: 'anymore' is redundant

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: remove "you" and "your"

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: how about

> See [Pipeline customization](/nservicebus/pipeline/customizing-v6.md).

Comment (HEskandari) [inline nservicebus/upgrades/5to6.md]: @SimonCropp That was fast :) updated!

File: nservicebus/upgrades/5to6.md [hunk 2]
```diff
@@ -241,6 +241,9 @@ The `IMessageMutator` and `IMutateTransportMessages` interfaces are no longer av
  * `IncomingContext` and `OutgoingContext` have been replaced with multiple contexts ("stages") per pipeline offering more useful information depending on the requirements. The [Version 6 guide on pipeline customization](/nservicebus/pipeline/customizing-v6.md) provides more details about the available stages.
  * `Behavior<TContext>` now provides the base class for implementing custom behaviors and replaces `IBehavior<TContext>`.
 
+### PipelineExecutor
+The `PipelineExecutor` has been removed. It has been internalized and is not accessible through the Container. You can use a custom Behavior instead. See [Pipeline customization](/nservicebus/pipeline/customizing-v6.md).
```
Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: remove you.

For example 

> Use a custom Behavior instead

Comment (SimonCropp) [inline nservicebus/upgrades/5to6.md]: link  Container to `/nservicebus/containers/`

**Pull request merged**